app/main.ipynb

```diff
@@ -56,33 +59,57 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 5,
-   "outputs": [],
    "source": [
-    "# write your code here (for Python default syntax)"
+    "arr = [0] * 1000\n",
+    "sys.getsizeof(arr)"
    ],
    "metadata": {
     "collapsed": false,
     "ExecuteTime": {
-     "end_time": "2023-07-03T12:02:41.673421Z",
-     "start_time": "2023-07-03T12:02:41.666277Z"
+     "end_time": "2024-11-07T17:44:05.996929Z",
+     "start_time": "2024-11-07T17:44:05.878009Z"
     }
-   }
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "8056"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "execution_count": 4
   },
   {
    "cell_type": "code",
-   "execution_count": 4,
-   "outputs": [],
    "source": [
-    "# write your code here (for numpy syntax)"
+    "np_array = np.array(arr)\n",
+    "sys.getsizeof(np_array)"
```

Comment:
The numpy array 'np_array' is created from a Python list, which does not minimize memory usage. Consider using 'np.zeros(1000, dtype=np.int8)' to create a numpy array with minimal memory usage as required by the task.